### PR TITLE
Update add-route.md

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by tessamero** [comment]
line 109 is originally fine. It should be realtime” and not “real time” as we are using that word as a noun and not an adjective as it’s the name of the subscribe method

Cannot approve

File: docs/tutorials/add-route.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ App::post('/v1/storage/buckets/:bucketId/files')
 ```
 
 #### Events
-* event - A pattern that is associated with the route in behalf of realtime messaging.
+* event - A pattern that is associated with the route in behalf of real time messaging.
```
Comment (gewenyu99) [inline docs/tutorials/add-route.md:109]: We consider `realtime` the correct spelling in our code base :)

**Review by gewenyu99 — DISMISSED** [review]

File: docs/tutorials/add-route.md (hunk 1 above)
Comment (tusharxoxoxo) [reply docs/tutorials/add-route.md]: reverted back

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Comment by tusharxoxoxo** [comment]
I don't know why but this email was automatically move to spam, today only
I was able to see it. By now the invitation link for joining Appwrite team
has expired, but I was very interested in joining the team. Can you please
resend me the invitation link. I would be very thankful to you.
Tushar

**Pull request merged**